packages/ui/src/ui-component/dialog/ExpandTextDialog.jsx

```diff
@@ -100,15 +100,15 @@ const ExpandTextDialog = ({ show, dialogProps, onCancel, onConfirm }) => {
                                     borderColor: theme.palette.grey['500'],
                                     borderRadius: '12px',
                                     height: '100%',
-                                    maxHeight: languageType === 'js' ? 'calc(100vh - 250px)' : 'calc(100vh - 220px)',
+                                    maxHeight: languageType === 'js' ? 'calc(100vh - 430px)' : 'calc(100vh - 400px)',
```

Comment:
actually prefer to have `languageType === 'js' ? 'calc(100vh - 330px)' : 'calc(100vh - 220px)'`

so we have more spaces in the editable area:
![image](https://github.com/FlowiseAI/Flowise/assets/26460777/115f5e84-f645-4775-824f-be5ac47193b9)

IMO `430` is lil bit small for edtiable area:
![image](https://github.com/FlowiseAI/Flowise/assets/26460777/f8539843-f5fe-4ac9-91fc-723c0c69799c)

Reply:
Sounds good to me, as long as the response is still visible.

Ideally the code view should adapt in size to cover as much as possible depending on the rest of the content that is shown, but my frontend knowledge is not good enough to do it 😅

Reply:
yeah thats good enough for now! thanks!